src/main/java/no/ndla/taxonomy/domain/GradeAverage.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Part of NDLA taxonomy-api
+ * Copyright (C) 2024 NDLA
+ *
+ * See LICENSE
+ */
+
+package no.ndla.taxonomy.domain;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.io.Serializable;
+import java.util.Collection;
+import java.util.Optional;
+
+public class GradeAverage implements Serializable {
```

Comment:
Må du ha med serializable her? Ser ut som om det blir med over i typescript-koden. 
I tillegg så eksponerer du her en domeneklasse i apiet. Tipper det er bedre å legge til en dto og så konvertere mellom dei.

Reply:
For å være ærlig så vet jeg ikke. Kan teste litt :smile: